mne/io/snirf/tests/test_snirf.py

```diff
@@ -238,11 +259,12 @@ def test_snirf_nirsport2_w_positions():
     # nirsite https://github.com/mne-tools/mne-testing-data/pull/86
     # figure 3
     allowed_distance_error = 0.005
-    distances = source_detector_distances(raw.info)
-    assert_allclose(distances[::2][:14],
```

Comment:
As the ordering of the channels is now changed, I test specific source detector pairs, rather than an indexed channel.